superset-frontend/src/views/CRUD/data/database/DatabaseList.tsx

```diff
@@ -171,8 +182,43 @@ function DatabaseList({ addDangerToast, addSuccessToast }: DatabaseListProps) {
   const canExport =
     hasPerm('can_export') && isFeatureEnabled(FeatureFlag.VERSIONED_EXPORT);
 
+  const uploadDropdownMenu = [
```

Comment:
since this is very similar to the previous menu list, is there anyway to combine them? If not that's obviously totally ok.

Reply:
I guess we could  create a function or some kind of hook that returns this array with the vars passed in. I think this will be good to do in follow up as we will need add in your changes for the check if user can upload.